src/classes/jsonld/class-jsonld-article-wrapper.php

```diff
@@ -114,7 +114,7 @@ public function after_get_jsonld( $jsonld, $post_id, $context ) {
 		 * for Person and Organization, so check before we add it to graph.
 		 * reference : https://schema.org/author
 		 */
-		if ( $author_jsonld && ! $this->is_author_entity_present_in_graph( $jsonld, $article_jsonld['author']['@id'] ) ) {
+		if ( $author_jsonld && ! $this->is_author_entity_present_in_graph( $jsonld, $this->get_primary_author($article_jsonld['author'])['@id'] ) ) {
```

Comment:
We need to be careful about this syntax as I think older version of PHP do not support it.

Let's change it to:

```
$primary_author = $this->get_primary_author($article_jsonld['author']);

if ( $author_jsonld && ! $this->is_author_entity_present_in_graph( $jsonld, $primary_author['@id'] ) ) {
```